platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/request/AjaxRequestMessage.ts

```diff
@@ -128,6 +144,22 @@ class AjaxRequestMessage {
 		}
 	}
 
+	private isSameDeltaDirections(other: AjaxRequestMessage) {
+		let thisDeltaX = Number(this.data.get('deltaX') ?? 0);
+		let otherDeltaX = Number(other.data.get('deltaX') ?? 0);
+		let deltaXSameSign = (thisDeltaX == 0 && otherDeltaX == 0) || thisDeltaX * otherDeltaX > 0;
```

Comment:
how about
```suggestion
		let thisDeltaX = Number(this.data.get('deltaX') ?? 0);
		let otherDeltaX = Number(other.data.get('deltaX') ?? 0);
		let deltaXSameSign = Math.sign(thisDeltaX) == Math.sign(otherDeltaX);
```

Reply:
looks fine, assuming that the distinction between positive and negative zero makes no difference here. cf https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sign